src/Routes.tsx

```diff
@@ -15,7 +15,7 @@ export const Routes: ReactNode = () => (
     }
   >
     <Switch>
-      <Route path="/satellite-manifest" component={SatelliteManifestPage} />
+      <Route path="/" component={SatelliteManifestPage} />
```

Comment:
Locally, all paths are leading me to the manifests page (`/anything`).  Shouldn't only `/` lead me there.